### PR TITLE
Change display of identical motions

### LINKED ISSUE #3418: Change display of 'identical motions'-list
Pre requirments: 
Two motions/amendments need to have identical text 

**Current display:**

**Wanted display:**
- If the motion/amendment has a number (Bezeichner) then the number should be displayed (otherwise the title) 
- The displayed numbers/titles should not each have their own line

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #3418

### CODE REVIEW

**Review by Elblinator — CHANGES_REQUESTED** [review]
please add a comma in between the different motions/amendments

**Comment by reiterl** [comment]
Who did say something about commas in the issue? No one.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html [hunk 1]
```diff
@@ -293,7 +293,8 @@ <h4>{{ (motion.hasLeadMotion ? 'Identical amendments' : 'Identical motions') | t
             [routerLink]="['/', activeMeetingId, 'motions', identicalMotion.sequential_number]"
             [state]="{ back: 'true' }"
         >
-            {{ identicalMotion.numberOrTitle }}
+            {{ identicalMotion.numberOrTitle
+            }}{{ showIdenticalMotionComma(identicalMotion, motion?.identical_motions) ? ',' : '' }}
```
Comment (jsangmeister) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html:297]: Please take a look at the `CommaSeparatedListingComponent` and exactly copy the behaviour from there:

https://github.com/OpenSlides/openslides-client/blob/1d78752728118b54fce22a62fea235955bd8139e/client/src/app/ui/modules/comma-separated-listing/components/comma-separated-listing/comma-separated-listing.component.html#L1-L8

You can use the `let isLast = last` syntax in combination with `ngIf` to determine whether or not to place a comma. Also, use the same special space ( `&ensp;`) to add spacing between the items.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html (hunk 1 above)
Comment (jsangmeister) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html:297]: Nevermind, you can simply use the component directly in combination with an `ng-template` - I overlooked this earlier, this is way better than copying the behaviour

Comment (jsangmeister) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html]: See my second comment

Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html]: Yes, using the component is a better idea. Add a commit with the `comma-seperated-listing-component`.

**Review by jsangmeister — APPROVED** [review]

**Review by Elblinator — DISMISSED** [review]
For the first motion in the list there is one space too much after the motion name
For the last motion there is one space too much before the motion name
The the motions in between there are two spaces to much, one before one after the motion name
Screenshot_20240314_104740

**Comment by reiterl** [comment]
Nice, I don't use the broken comma separated listing anymore. Please retest.

**Review by jsangmeister — CHANGES_REQUESTED** [review]
I would much rather have solution using the listing component fixed. It looks to me like the additional spaces around the number in the html are creating this behaviour, you could try removing them in the template, wrapping the number in a separate `span` or implement a fix inside the component.

**Comment by reiterl** [comment]
@jsangmeister Please give a code suggestion!

**Comment by jsangmeister** [comment]
Fixed the usage of the component by wrapping the text in an additional `span` which removed the additional whitespaces.

**Review by jsangmeister — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
The layout should be improved:
* use midpoints instead of comma
* move new metadata block above category (see arrow in screenshot)
image

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
see comment

**Comment by reiterl** [comment]
I have added a midpoint option to the comma separated listing and use it here and moved the metadata above categories metadata.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html [hunk 2]
```diff
@@ -285,19 +297,6 @@ <h4>{{ 'Origin' | translate }}</h4>
     </div>
 </div>
 
-<!-- Identical motions -->
-<div *ngIf="motion?.identical_motions?.length">
-    <h4>{{ 'Identical with' | translate }}</h4>
```
Comment (emanuelschuetze) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html:290]: Please keep this changed string.

**Comment by emanuelschuetze** [comment]
One (string) remark. Midpoind and position is fine now.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html (hunk 2 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html:290]: Renamed the label.

**Review by bastianjoel — APPROVED** [review]

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**